### PR TITLE
fix: UI of Send Activity

### LINKED ISSUE #307: It's  should be "Amount" instead of "Label" in Send Fragment
**Summary:**

It's should be "Amount" instead of "Label" in Send Fragment, which is more user-friendly and apt

**Device and Android version:**

Realme 2 Pro
Android version 8.1.0

**Screenshots:**

screenshot_2019-02-14-19-52-20-30 1

### PR DESCRIPTION
**Pull request by shaliniv16**
Fixes #307 

**Screenshots**

screenshot_2019-02-16-17-14-13-34

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

### CODE REVIEW

**Comment by shaliniv16** [comment]
@luckyman20 please review.

File: mifospay/src/main/res/layout/fragment_send.xml [hunk 1]
```diff
@@ -18,7 +18,9 @@
         android:layout_marginBottom="0dp"
         android:orientation="vertical"
         tools:layout_editor_absoluteX="0dp"
-        tools:layout_editor_absoluteY="0dp">
+        tools:layout_editor_absoluteY="0dp"
+        android:layout_alignParentLeft="true"
```
Comment (luckyman20) [inline mifospay/src/main/res/layout/fragment_send.xml]: What are these changes for? I do not see any UI change after these.

Comment (shaliniv16) [reply mifospay/src/main/res/layout/fragment_send.xml]: @luckyman20 To support older version than API 17 I made these changes.

Comment (naman14) [reply mifospay/src/main/res/layout/fragment_send.xml]: @shaliniv16 can you explain how this affects older api versions

File: mifospay/src/main/res/layout/fragment_send.xml [hunk 2]
```diff
@@ -97,7 +99,7 @@
                     android:id="@+id/et_amount"
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
-                    android:hint="Label"
+                    android:hint="Amount"
```
Comment (naman14) [inline mifospay/src/main/res/layout/fragment_send.xml]: extract to strings.xml

**Pull request closed without merging**